examples/README.txt

```diff
@@ -6,10 +6,9 @@ Examples
 End-to-end examples show how you can use PyFluent. If the PyFluent ``ansys-fluent-core`` package is installed on your machine,
 you can download these examples as Python files or Jupyter notebooks and run them locally.
 
-A collection of PyFluent examples can be found at `pyfluent-examples <https://github.com/ansys/pyfluent-examples>`_.
-
 .. toctree::
    :maxdepth: 1
    :hidden:
 
-.. 00-fluent/README.txt
\ No newline at end of file
+   00-fluent/README.txt
+   Miscellaneous Examples <https://examples.fluent.docs.pyansys.com/version/dev/examples/index.html>    
```

Comment:
Minor suggestion: would "additional" examples rather than "miscellaneous" work better here? The examples linked are as "miscellaneous" as the ones in the core repo (i.e., there is not only a single specific type of example in the core repo, they are also varied). But these are "additional" examples being linked.
```suggestion
   Additional Examples <https://examples.fluent.docs.pyansys.com/version/dev/examples/index.html>    
```

Reply:
Yeah, Let's commit as Miscellaneous Examples for now, will update it in future and there might be chances will introduce more sections to page. Thanks!